user-guide/dataminer_services/Admin/Managing_dataminer_services_keys.md

```diff
@@ -51,7 +51,7 @@ Each set of keys consists of a (user-defined) label, a primary key, and a second
 - To **copy** a primary or secondary key, click the copy icon next to the key.
 - To **add** a new set of keys, click *New Key* at the top of the page, and specify a label.
   
-  When adding a new key, you have to specify permissions for the key. The *Register catalog items* permission will allow the key to be used to [register new Catalog items](xref:Register_Catalog_Item). The *Add Dataminer nodes* permission will allow the key to be used to add new DataMiner nodes, e.g. when creating a DataMiner System with the [pre-installed DataMiner Virtual Hard Disk](xref:Using_a_pre_installed_DataMiner_Virtual_Hard_Disk).
+  When adding a new key, you have to specify permissions for the key. The *Register catalog items* permission will allow the key to be used to [register new Catalog items](xref:Register_Catalog_Item). The *Add Dataminer System* permission will allow the key to be used to add new DataMiner Systems, e.g. when creating a DMS with the [pre-installed DataMiner Virtual Hard Disk](xref:Using_a_pre_installed_DataMiner_Virtual_Hard_Disk).
```

Comment:
Could this permission be adjusted to the correct capitalization as well (i.e. DataMiner instead of Dataminer)? I assume that the current capitalization is used to mirror the UI, but since we're adjusting this now, perhaps now is a good moment to immediately get it right completely?

Reply:
The capitalization has been corrected :)